iocage/Pkg.py

```diff
@@ -493,12 +492,3 @@ def __mount_pkg_directory(self, jail: iocage.Jail.JailGenerator) -> None:
             pass
         finally:
             jail.fstab.save()
-
-    def __umount_pkg_directory(self, jail: iocage.Jail.JailGenerator) -> None:
```

Comment:
I'd keep the function around, might be useful to creating small surface containers